src/test/java/com/google/api/generator/gapic/composer/GrpcServiceCallableFactoryClassComposerTest.java

```diff
@@ -35,6 +38,10 @@
 public class GrpcServiceCallableFactoryClassComposerTest {
   private ServiceDescriptor echoService;
   private FileDescriptor echoFileDescriptor;
+  private static final String GOLDENFILES_DIRECTORY =
+      "src/test/java/com/google/api/generator/gapic/composer/goldens/";
+  private static final String GOLDENFILES_NAME =
+      "GrpcServiceCallableFactoryClassComposerTest.golden";
```

Comment:
For the `*ClassComposerTest`s, can we give these the same name as the generated class?

Reply:
Sure, renamed the goldens as the generated class name.